client/settings/express-checkout-settings/platform-checkout-settings.js

```diff
@@ -66,10 +67,57 @@ const PlatformCheckoutSettings = ( { section } ) => {
 						checked={ isPlatformCheckoutEnabled }
 						onChange={ updateIsPlatformCheckoutEnabled }
 						label={ __( 'Enable WooPay', 'woocommerce-payments' ) }
-						help={ __(
-							'When enabled, customers will be able to checkout using WooPay',
-							'woocommerce-payments'
-						) }
+						help={
+							/* eslint-disable jsx-a11y/anchor-has-content */
+							isPlatformCheckoutEnabled
+								? __(
+										'Boost conversion and customer loyalty by offering a single click, secure way to pay.',
+										'woocommerce-payments'
+								  )
+								: interpolateComponents( {
+										mixedString: __(
+											/* eslint-disable-next-line max-len */
+											'Boost conversion and customer loyalty by offering a single click, secure way to pay. ' +
+												'In order to use {{wooPayLink}}WooPay{{/wooPayLink}}, you must agree to our ' +
+												'{{tosLink}}WooCommerce Terms of Service{{/tosLink}} ' +
+												'and {{privacyLink}}Privacy Policy{{/privacyLink}}. ' +
+												'{{trackingLink}}Click here{{/trackingLink}} to learn more about the ' +
+												'data you will be sharing and opt-out options.',
+											'woocommerce-payments'
+										),
+										components: {
+											wooPayLink: (
+												<a
+													target="_blank"
+													rel="noreferrer"
+													href="https://woocommerce.com/document/woopay-merchant-documentation/"
+												/>
+											),
```

Comment:
I haven't seen this link in the legal mandate copy attached to the PR description. It's probably coming from the previous copy where WooPay was hyperlinked. Could we please confirm that the link is needed?

Reply:
@FangedParakeet lacking an easy way to access documentation on the settings page, I'd leave it linked for now. The parent page has the "learn more" link for EPMs while being deep linked into settings discoverability of documentation becomes harder.